src/lib.rs

```diff
@@ -354,16 +354,21 @@ impl Model for () {
 
 // Helpers for stable hashing, wherein hashes should not vary across builds.
 mod stable {
+    use std::hash::BuildHasher;
+
     use ahash::{AHasher, RandomState};
 
     const KEY1: u64 = 123_456_789_987_654_321;
     const KEY2: u64 = 98_765_432_123_456_789;
+    // TODO: how to get these?
+    const KEY3: u64 = 0;
+    const KEY4: u64 = 0;
```

Comment:
My take is that any value is fine as long as it's consistent before the library hits 1.0.